akka-http/src/main/scala/akka/http/javadsl/unmarshalling/sse/EventStreamUnmarshalling.scala

```diff
@@ -8,21 +8,30 @@ package unmarshalling
 package sse
 
 import akka.NotUsed
+import akka.actor.ActorSystem
 import akka.http.javadsl.model.HttpEntity
 import akka.http.javadsl.model.sse.ServerSentEvent
 import akka.stream.javadsl.Source
 
 /**
- * Using `fromEventStream` lets a `HttpEntity` with a `text/event-stream` media type be unmarshalled to a source of
+ * Using `fromEventsStream` lets a `HttpEntity` with a `text/event-stream` media type be unmarshalled to a source of
  * `ServerSentEvent`s.
  */
 object EventStreamUnmarshalling {
 
   /**
-   * Lets a `HttpEntity` with a `text/event-stream` media type be unmarshalled to a source of `ServerSentEvent`s.
+   * Lets an `HttpEntity` with a `text/event-stream` media type be unmarshalled to a source of `ServerSentEvent`s.
    */
+  def fromEventsStream(implicit system: ActorSystem): Unmarshaller[HttpEntity, Source[ServerSentEvent, NotUsed]] =
+    scaladsl.unmarshalling.sse.EventStreamUnmarshalling.fromEventsStream(system)
+      .map(_.map(_.asInstanceOf[ServerSentEvent]).asJava)
+      .asInstanceOf[Unmarshaller[HttpEntity, Source[ServerSentEvent, NotUsed]]]
+
+  // for binary-compatibility, since 10.0.15
+  @deprecated("Binary compatibility method. Invocations should have an implicit ActorSystem in scope to provide access to configuration", "10.0.15")
```

Comment:
I guess this should be 'deprecated since 10.1.6'?

Reply:
```suggestion
  @deprecated("Binary compatibility method. Invocations should have an implicit ActorSystem in scope to provide access to configuration", "10.1.6")
```